### PR TITLE
Add Crystal topic

### PR DESCRIPTION
**Pull request by faustinoaq**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [x] Curating a new Topic page

***********EDITING AN EXISTING TOPIC PAGE************

I'm suggesting these edits to an existing topic:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:

This PR adds information for `crystal` topic related to [The Crystal Programming Language](https://crystal-lang.org). Currently exists [2000+](https://crystalshards.xyz) projects on Github using this language and [500+](https://github.com/topics/crystal) repositories using `crystal` topic.

In conclusion, [Crystal is preparing for first stable release](https://crystal-lang.org/2016/12/29/crystal-new-year-resolutions-for-2017-1-0.html), so this PR will help to give useful information about this new programming language.

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/crystal/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+created_by: Ary Borenszweig
+display_name: Crystal
+logo: crystal.png
+released: June 2011
+short_description: Crystal is a self-hosted, general-purpose programming language.
+topic: crystal
+url: https://crystal-lang.org/
+wikipedia_url: https://en.wikipedia.org/wiki/Crystal_(programming_language)
+---
+Crystal is a programming language with friendly syntax, static type checking and modern stdlib. It compiles into efficient, easy to distribute native code.
```
Comment (nayafia) [inline topics/crystal/index.md]: `easy to distribute` -> `easy-to-distribute`

File: topics/crystal/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+aliases: crystal-lang
+created_by: Ary Borenszweig
+display_name: Crystal
+logo: crystal.png
+released: June 2011
+short_description: Crystal is a self-hosted, general-purpose programming language.
```
Comment (nayafia) [inline topics/crystal/index.md]: `general-purpose` -> `general purpose`

File: topics/crystal/index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+aliases: crystal-lang
+created_by: Ary Borenszweig
```
Comment (nayafia) [inline topics/crystal/index.md]: Looking at Crystal's Wikipedia page, should `Juan Wajnerman` also be listed as a creator?

File: topics/crystal/index.md (hunk 1 above)
Comment (faustinoaq) [reply topics/crystal/index.md]: Done! 👍

File: topics/crystal/index.md (hunk 2 above)
Comment (faustinoaq) [reply topics/crystal/index.md]: Done! 👍

File: topics/crystal/index.md (hunk 3 above)
Comment (faustinoaq) [reply topics/crystal/index.md]: Done! 👍

**Comment by faustinoaq** [comment]
I did changes according to suggestions you commented 👍

**Comment by cheshire137** [comment]
@nayafia, faustinoaq made some changes since your last review. 👀

**Comment by nayafia** [comment]
Thank you @faustinoaq! 💖
